### PR TITLE
Feature/signup layout

### PR DESCRIPTION
**Pull request by izabelrosen**
Please review sign up form, go to: /sign-up

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Great job with the responsive and layout.

**Comment by izabelrosen** [comment]
After some modification on the pages layout it is officially done - please review!

File: src/Components/Forms/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react'
+import JWT from 'jsonwebtoken'
+import Form from 'react-jsonschema-form'
+import Alert from '../Alert'
+import Axios from '../../Lib/Common/Axios'
```
Comment (SebastianGerS) [inline src/Components/Forms/SignUp.js]: Axios is not used so it should be removed (even if we might ad it in later)

**Review by SebastianGerS — APPROVED** [review]
Looks good!

just remove the Axios import from SignUp.js and we are good to go!

**Comment by izabelrosen** [comment]
Cleaned up unused imports.

**Comment by izabelrosen** [comment]
Conflicts solved - ready for merge.

**Pull request merged**